pkg/release/config/config.go

```diff
@@ -34,12 +34,13 @@ type Job struct {
 }
 
 type AggregatedJob struct {
-	AnalysisJobCount int `json:"analysisJobCount"`
+	ProwJob          *Job `json:"prowJob,omitempty"`
```

Comment:
Do we have an example of this in https://github.com/openshift/release/tree/master/core-services/release-controller/_releases?

Reply:
Not at this moment. The codes support it. So if anyone adds the config, this should be support by my change.

Reply:
>Not at this moment. The codes support it.

Do you mean the code of release controllers? Could you please link?

Reply:
Yes, release controller. Here is the line: https://github.com/openshift/release-controller/blob/e6e5923afde7a61bd763c3f1b3e0e3bef0ab4e9a/cmd/release-controller/sync_verify_prow.go#L73